L1Trigger/L1THGCal/python/hgcalVFEProducer_cfi.py

```diff
@@ -30,6 +30,12 @@
 fCPerMIP_200 = fCPerMIPee[1]
 thicknessCorrection_200 = thicknessCorrection[1]
 
+# NOSE parameters
+fCPerMIPnose = recoparam.HGCalUncalibRecHit.HGCHFNoseConfig.fCPerMIP
+layerWeightsNose = recocalibparam.dEdX.weightsNose
+##thicknessCorrectionNose = recocalibparam.HGCalRecHit.thicknessNoseCorrection
```

Comment:
delete commented-out code

Reply:
Hardcoded lines 37 
thicknessCorrectionNose = [0.759,0.760,0.773]
should be removed and line 36 uncommented